pkg/virt-api/webhooks/mutating-webhook/mutators/clone-create-mutator_test.go

```diff
@@ -38,8 +39,7 @@ var _ = Describe("Clone mutating webhook", func() {
 		Expect(err).ToNot(HaveOccurred())
 		Expect(patch).NotTo(BeEmpty())
 
-		Expect(cloneSpec.Target).ShouldNot(BeNil())
-		Expect(cloneSpec.Target.Name).ShouldNot(BeEmpty())
+		Expect(cloneSpec.Target).To(Equal(newExpectedTarget(vmClone.Spec.Source)))
```

Comment:
I think it will be better to inline the expected target, so it will be visible to the reader directly.

Reply:
Done.